Lib/distutils/util.py

```diff
@@ -90,8 +92,25 @@ def get_platform ():
 
     return "%s-%s-%s" % (osname, release, machine)
 
-# get_platform ()
+# get_host_platform ()
+
+def get_platform ():
```

Comment:
The space after the function name isn't correct style (anymore) - update anything you touch to not include it.